ui/widgets/nuxeo-user-avatar.js

```diff
@@ -296,36 +327,28 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
             '9',
             '0',
           ];
-          let tempName = '';
-          const splitName = name.split(' ');
+
           const alphabetPosition = alphabet.indexOf(name.charAt(0).toLowerCase());
-          this.$.container.style.backgroundColor = colors[alphabetPosition];
-          for (let i = 0; i < splitName.length; i++) {
-            tempName += splitName[i].charAt(0);
-          }
-          this._output = tempName;
-          if (this.fetchAvatar) {
-            this.$.getUserProfile.path = `user/${this._username(this.user)}`;
-            this.$.getUserProfile
-              .get()
-              .then((res) => {
-                if (
-                  res.contextParameters &&
-                  res.contextParameters.userprofile &&
-                  res.contextParameters.userprofile.avatar
-                ) {
-                  this.user = res;
-                }
-              })
-              .catch(() => {
-                console.warn(`Cannot fetch profile for user ${this._username(this.user)}`);
-              });
+          if (alphabetPosition === -1) {
+            // ELEMENTS-903: If case user's data contains non Latin characters show icon
```

Comment:
We already have this comment on the git history, with JIRA reference.
We should avoid comments on code unless it is something hacky or that needs special attention. In that kind of situations I believe the convention is `// XXX - something that needs our attention!`